examples/gno.land/r/demo/profile/profile.gno

```diff
@@ -53,42 +53,49 @@ var boolFields = map[string]bool{
 
 // Setters
 
-func SetStringField(field, value string) error {
+func SetStringField(field, value string) bool {
 	addr := std.PrevRealm().Addr()
-	if _, ok := stringFields[field]; !ok {
-		return errors.New("invalid string field")
+	key := addr.String() + ":" + field
```

Comment:
Why did you remove the allowed fields check?

Reply:
the original intent of this realm was to allow arbitrary fields

> The goal is to pack the contract with default fields that are the expected most common ones for customization (profile_picture, name, primary_color, is_dev (to enable more debug), etc. But the contract should support a way to set new fields, basically a key/value store associated with a profile.

source: https://github.com/gnolang/gno/pull/181#issue-1229907667

I don't know why the check are there in the first place, we looked through the conversations in the initial PR and we found no rationale for these checks